### PR TITLE
View all grades

### PR DESCRIPTION
**Pull request by helen-choi**
Screen Shot 2020-04-18 at 1 15 09 AM

### CODE REVIEW

File: server/public/styles.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.container {
+  display: flex;
+  flex-wrap:wrap;
+  width: 100%;
+  max-width: 1250px;
+  height: 100vh;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: Bootstrap already defines a `container` class. But it also defines some flex helpers. See if you can use those to reduce the amount of custom CSS required.

https://getbootstrap.com/docs/4.4/utilities/flex/

File: server/public/index.html [hunk 2]
```diff
@@ -6,17 +6,28 @@
...
 <body>
-  <div id="root"></div>
+  <div id="root">
+    <div class="container">
+      <div class="list">
+        <h1>Student Grade Table</h1>
+
+      </div>
+      <div class="form"></div>
+    </div>
```
Comment (thebearingedge) [inline server/public/index.html:27]: You can remove these things since React will erase them anyway.

File: server/public/index.html [hunk 3]
```diff
@@ -6,17 +6,28 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>SGT: React</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
```
Comment (thebearingedge) [inline server/public/index.html]: Bootstrap is already linked for you.

File: server/public/index.html [hunk 4]
```diff
@@ -6,17 +6,28 @@
...
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   <link rel="shortcut icon" type="image/png" href="/images/favicon.png">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.4.1/css/bootstrap.min.css"
     integrity="sha256-L/W5Wfqfa0sdBNIKN9cG6QA5F2qx4qICmU2VgLruv9Y=" crossorigin="anonymous" />
   <link rel="stylesheet" crossorigin="anonymous"
     href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css"
     integrity="sha256-46qynGAkLSFpVbEBog43gvNhfrOj+BmwXdxFgVK/Kvc=" />
-  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```
Comment (thebearingedge) [inline server/public/index.html:15]: Bootstrap is already linked for you.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: server/public/styles.css [hunk 5]
```diff
@@ -0,0 +1,10 @@
+/* .container {
+  display: flex;
+  flex-wrap:wrap;
+  width: 100%;
+  max-width: 1250px;
+  height: 100vh;
+} */
```
Comment (thebearingedge) [inline server/public/styles.css]: Please do not turn in commented out code 😢

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: server/public/styles.css (hunk 5 above)
Comment (helen-choi) [reply server/public/styles.css]: Oops! I meant to delete :)

**Pull request merged**